share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -2,6 +2,8 @@
 # Copyright (C) 2017 Advanced Micro Devices, Inc.
 # ######################################################################################################################
 
+cmake_minimum_required( VERSION 3.6 )
```

Comment:
I think it would be better to just set the policy needed.

Reply:
Changed to only set CMP0057 to NEW.